src/runtime/server/storage.ts

```diff
@@ -144,16 +147,14 @@ export const getContentsList = (() => {
     if (event.context.__contentList) {
       return event.context.__contentList
     }
-    if (isPrerendering && cachedContents.length) {
+    if (cachedContents.length) {
```

Comment:
Shall we keep `isPrerendering` and add a condition for dev?

```suggestion
    if ((isPrerendering || !isProduction) && cachedContents.length) {
```